README.md

```diff
@@ -74,44 +51,39 @@ Kubernetes directly.
 **Configuration 3:**
 ![screenshot](./images/gke-deployment.png)
 
-A simple Python Flask web application (Prime-flask) was created for this
-demonstration which contains two endpoints:
+A simple Python [Flask](http://flask.pocoo.org/) web application (`Prime-flask`) was created for this demonstration which contains two endpoints:
 
 `http://[ip]:8080/factorial/` and
 
 `http://[ip]:8080/prime/`
 
-examples would look like:
+Examples of use would look like:
 
 ```console
-curl http://[ip]:8080/prime/10
+curl http://35.227.149.80:8080/prime/10
 The sum of all primes less than 10 is 17
 
-curl http://[ip]:8080/factorial/10
+curl http://35.227.149.80:8080/factorial/10
```

Comment:
as above

Reply:
Ultimately, as the reviewer you get to decide. Lemme know if you find my argument convincing or if you'd like to discuss more!